PWGCF/FemtoDream/femtoDreamPairTaskTrackTrack.cxx

```diff
@@ -182,6 +218,10 @@ struct femtoDreamPairTaskTrackTrack {
           continue;
         }
         trackHistoPartTwo.fillQA(part);
+        if constexpr (isMC) {
+          trackHistoPartTwoMC.fillQA<true>(partsMC.iteratorAt(part.index()));
+          trackHistoPartTwoMC.fillQAMC(part, partsMC.iteratorAt(part.index()));
+        }
```

Comment:
Keeping asking along the whole code if it is MC or not instead of having it centralized on the (templated) function that you call is really really error prone
Please, reconsider it